erpnext/accounts/doctype/account/account.py

```
@@ -162,113 +424,202 @@ def validate_mandatory(self):
-							fmt_money(stock_balance, self.account_currency)))
 
-		elif self.warehouse:
-			self.warehouse = None
+@frappe.whitelist()
+@frappe.validate_and_sanitize_search_inputs
+def get_parent_account(doctype, txt, searchfield, start, page_len, filters):
+	return frappe.db.sql(
+		"""select name from tabAccount
+		where is_group = 1 and docstatus != 2 and company = {}
+		and {} like {} order by name limit {} offset {}""".format("%s", searchfield, "%s", "%s", "%s"),
+		(filters["company"], "%%%s%%" % txt, page_len, start),
```

Ruff (0.8.2)

Use format specifiers instead of percent format

Replace with format specifiers

(UP031)